x-pack/plugins/security_solution/public/common/components/import_data_modal/index.tsx

```diff
@@ -69,41 +73,65 @@ export const ImportDataModalComponent = ({
   const [isImporting, setIsImporting] = useState(false);
   const [overwrite, setOverwrite] = useState(false);
   const [overwriteExceptions, setOverwriteExceptions] = useState(false);
+  const [overwriteActionConnectors, setOverwriteActionConnectors] = useState(false);
   const { addError, addSuccess } = useAppToasts();
-
+  // TODO Ask if we use schema on the FE
```

Comment:
Yeah, so long as they're in `common` it's all good. You'll see a bunch of FE usages from the schemas in the other `common/detection_engine` dirs.

Reply:
Awesome thanks!